README.md

```diff
@@ -1,29 +1,33 @@
 # go-mysql
 
-A pure go library to handle MySQL network protocol and replication.
+A pure go library to handle MySQL network protocol and replication as used by MySQL and MariaDB.
 
 ![semver](https://img.shields.io/github/v/tag/go-mysql-org/go-mysql)
 ![example workflow](https://github.com/go-mysql-org/go-mysql/actions/workflows/ci.yml/badge.svg)
 ![gomod version](https://img.shields.io/github/go-mod/go-version/go-mysql-org/go-mysql/master)
-
-## How to migrate to this repo
-To change the used package in your repo it's enough to add this `replace` directive to your `go.mod`:
-```
-replace github.com/siddontang/go-mysql => github.com/go-mysql-org/go-mysql v1.10.0
-```
-
-v1.10.0 - is the last tag in repo, feel free to choose what you want.
+[![Go Reference](https://pkg.go.dev/badge/github.com/go-mysql-org/go-mysql.svg)](https://pkg.go.dev/github.com/go-mysql-org/go-mysql)
 
 ## Changelog
 This repo uses [Changelog](CHANGELOG.md).
 
 ---
 # Content
-* [Replication](#replication)
-* [Incremental dumping](#canal)
-* [Client](#client)
-* [Fake server](#server)
-* [database/sql like driver](#driver)
+* [Replication](#replication) - Process events from a binlog stream.
+* [Incremental dumping](#canal) - Sync from MySQL to Redis, Elasticsearch, etc.
```

Comment:
IMO the difference between this "Incremental dumping" and "Replication" item are developers need to implement interfaces (like `EventHandler`) or handle decoded binlog events. Maybe use "library mode" or "framework mode". I choose these two words to tell the difference like 

> When you use a library, you are in charge of the flow of the application. You are choosing when and where to call the library. When you use a framework, the framework is in charge of the flow. It provides some places for you to plug in your code, but it calls the code you plugged in as needed.

https://www.freecodecamp.org/news/the-difference-between-a-framework-and-a-library-bd133054023f/ (a random blog ranks top in google search "library vs framework")

Reply:
Not sure what you want me to change. Do you you want to clarify the difference between "Replication" and "Incremental dumping"?

Reply:
yes, like

```
* [Use as a library](#replication) - Process events from a binlog stream and have maximum functionalities.
* [Use as a framework](#canal) - Implement `EventHandler` interface to subscribe binlog changes
```

Reply:
However I don't have confidence how to describe these two items. Will think about it later